### PR TITLE
fixtures: addition of licenses fixtures

### PR DESCRIPTION
**Pull request by krzysztof**
* Adds fixtures for legacy zenodo licenses.

### CODE REVIEW

File: zenodo/modules/fixtures/data/licenses.json [hunk 1]
```diff
@@ -0,0 +1,1451 @@
+[
```
Comment (lnielsen) [inline zenodo/modules/fixtures/data/licenses.json]: Doesn't many of these licenses already exist in the existing vocabulary from Invenio-OpenDefinition

Comment (krzysztof) [inline zenodo/modules/fixtures/data/licenses.json]: They do, I am just using original Zenodo licenses dump as and then ignore the already existing licenses.

File: zenodo/modules/fixtures/data/licenses_map.json [hunk 2]
```diff
@@ -0,0 +1,102 @@
+{
+  "bsd-3-clause": "BSD-3-Clause",
+  "bsd-2-clause": "BSD-2-Clause",
+  "afl-3.0": "AFL-3.0",
+  "apl1.0": "APL-1.0",
+  "agpl-v3": "AGPL-3.0",
+  "against-drm": "Against-DRM",
+  "apache2.0": "Apache-2.0",
+  "apache": "Apache-2.0",
```
Comment (krzysztof) [inline zenodo/modules/fixtures/data/licenses_map.json]: Should ``"apache": "Apache-1.1"`` ?

File: zenodo/modules/fixtures/data/licenses_map.json [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+  "ca-tosl1.1": "CATOSL-1.1",
+  "cc-by-sa": "CC-BY-SA-4.0",
+  "cc-by": "CC-BY-4.0",
+  "cc-zero": "CC0-1.0",
+  "dsl": "DSL",
+  "eudatagrid": "EUDatagrid",
+  "eclipse-1.0": "EPL-1.0",
+  "ecl2": "ECL-2.0",
+  "ver2_eiffel": "EFL-2.0",
+  "eiffel": "EFL-2.0",
```
Comment (krzysztof) [inline zenodo/modules/fixtures/data/licenses_map.json]: There doesn't seem to be ``EFL-1.0`` in OD.org

File: zenodo/modules/fixtures/data/licenses_map.json (hunk 2 above)
Comment (lnielsen) [inline zenodo/modules/fixtures/data/licenses_map.json]: No, i think it's fine like this.

**Comment by krzysztof** [comment]
I have added license matching function to CLI for future reference.

**Pull request merged**